d2l-test-runner.config.js

```diff
@@ -1,3 +1,6 @@
+import { env } from 'node:process';
+
 export default {
-	pattern: type => `**/test/*.${type}.js`
+	pattern: type => `**/test/*.${type}.js`,
+	testReporting: !!env['CI']
```

Comment:
This gets vdiff as well for now. since it's run from within a GitHub Action I didn't want to change the action to enable test reporting. This can go away/become the new default eventually so `BrightspaceUI/testing` always generates the reports when run in CI.